examples/azure/postgres/build.sh

```diff
@@ -31,12 +32,28 @@ server <= azure_postgres_set_max_size($server, $size)
 server <= azure_postgres_set_version($server, $version)
 
 if $ssl == "false" {
-        server <= azure_postgres_disable_ssl($server)
+	server <= azure_postgres_disable_ssl($server)
 }
 
 azure_postgres_server_create($server)
 
 echo "created Postgres Server"
+echo "creating Postgres Server Firewall rule"
+
+publicip, status <= curl ipinfo.io/ip
```

Comment:
o.o

Are you sure that's safe? Adding a rule in the firewall for an ip provided by a third party service? I understand that this is an example, but people will actually use it (I would like to trust the examples).